### DISCUSSION TITLE
How to read a .js/.ts file and get its default export (CLI)?

### DISCUSSION THREAD

**Discussion by mkosir** [oven-sh/bun#6345] [Q&A] [answered]
Hey all 👋 
wondering how it would be possible to get default export from a file in Bun, such as:

```ts
// myConfigFile.ts
const myValue = { name: 'test', value: 42 };

export default myValue;
```

CLI (separate) application:
```ts
// readFile.js
const getDefaultExportValue = () => {
  const file = Bun.file('./myConfigFile');
  ...
  return myValue;
}
```

**Comment by alexeyten** [comment, marked as answer]
Are you looking for a dynamic import? 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import
